Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1465,3 +1466,65 @@ void CFrame::ParseHotkeys()
 	if (IsHotkey(HK_UNDO_SAVE_STATE))
 		State::UndoSaveState();
 }
+
+void CFrame::HandleFrameSkipHotkeys()
+{
+	static const int MAX_FRAME_SKIP_DELAY = 60;
+	static int frameStep = 0;
+	static int frameStepDelay = 30;
+	static int skipFrameStepDelay = 1;
+	static int skipFrameStepDelayCount = 0;
```

Comment:
Also, could you rename these two so it's clearer what they do? Maybe something with `repeat` or `hold` instead of `skip`...